GDJS/scripts/install-spine.js

```diff
@@ -0,0 +1,14 @@
+const fs = require('fs');
+const path = require('path');
+
+const format = 'utf8';
+
+const originalSpineDir = path.resolve('node_modules/pixi-spine');
+const originalSpinePackage = JSON.parse(fs.readFileSync(path.join(originalSpineDir, 'package.json'), format));
+const originalSpinePath = path.join(originalSpineDir, originalSpinePackage.extensionConfig.bundle);
+const originalSpine = fs.readFileSync(originalSpinePath, format);
+
+const varSpineExport = '\nvar pixi_spine = this.PIXI.spine;\n';
+const runtimeSpinePath = 'Runtime/pixi-renderers/pixi-spine.js';
+
+fs.writeFileSync(runtimeSpinePath, originalSpine + varSpineExport, format);
```

Comment:
Nitpicking: we usually have scripts to output things in the console so that it's clear for a developer what's happening at installation:

```js
const shell = require('shelljs');

try { 
  shell.echo(`✅ Properly copied pixi-spine.js from node_modules.`)
} catch(error) {
  shell.echo(`❌ Unable to copy pixi-spine.js from node_modules. Error is: ${error}`)
  shell.exit(1);
}
```

Reply:
done